packages/api/src/external/carequality/document/shared.ts

```diff
@@ -50,6 +50,9 @@ export const cqToFHIR = (
     ...(docRef.creation ? { creation: docRef.creation } : {}),
   };
 
+  const containedResources = docRef.authorInstitution
+    ? generateCQFHIRContained(docRef.authorInstitution)
+    : undefined;
```

Comment:
I don't think my comment was addressed - if `docRef.authorInstitution` is undefined, we want to set the contained org to be the one we queried from the CQ directory entry, right? the org should never be undefined, since we know where we're requesting docs from